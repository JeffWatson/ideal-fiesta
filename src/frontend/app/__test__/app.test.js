import React from 'react';
import renderer from 'react-test-renderer';
import App from '../app';

describe('frontend/app', () => {
  it('renders correctly', () => {
    const app = renderer.create(<App />);
    expect(app).toMatchSnapshot();
  });
});

