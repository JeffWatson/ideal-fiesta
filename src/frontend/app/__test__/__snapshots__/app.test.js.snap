// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`frontend/app renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="App-header"
  >
    <h2>
      Welcome to Ideal Fiesta
    </h2>
  </div>
  <div
    className="matrix"
  >
    <div
      className="current-player-info"
    >
      The current Player is: 
      BLUE
    </div>
    <table
      className="matrix-table"
    >
      <tbody>
        <tr
          className="matrix-row"
        >
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div>
                <div
                  className="unit soldier green"
                >
                  Soldier!
                </div>
                <div
                  className="health-container"
                >
                  20
                </div>
              </div>
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div>
                <div
                  className="unit soldier red"
                >
                  SmallTank!
                </div>
                <div
                  className="health-container"
                >
                  20
                </div>
              </div>
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
        </tr>
        <tr
          className="matrix-row"
        >
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="ocean-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="ocean-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="ocean-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="ocean-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="land-factory-container"
              onClick={[Function]}
            >
              <div>
                <div
                  className="health-container"
                >
                  20
                </div>
              </div>
            </div>
          </td>
        </tr>
        <tr
          className="matrix-row"
        >
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="grass-container"
            >
              <div>
                <div
                  className="unit soldier blue"
                >
                  Soldier!
                </div>
                <div
                  className="health-container"
                >
                  20
                </div>
              </div>
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="mountain-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
        </tr>
        <tr
          className="matrix-row"
        >
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="grass-container"
            >
              <div>
                <div
                  className="unit soldier blue"
                >
                  SmallTank!
                </div>
                <div
                  className="health-container"
                >
                  20
                </div>
              </div>
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
        </tr>
        <tr
          className="matrix-row"
        >
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="grass-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div />
            </div>
          </td>
          <td
            className="matrix-cell"
            onClick={[Function]}
          >
            <div
              className="sand-container"
            >
              <div>
                <div
                  className="unit soldier yellow"
                >
                  Soldier!
                </div>
                <div
                  className="health-container"
                >
                  20
                </div>
              </div>
            </div>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
